docs/UserGuide.md

```diff
@@ -174,7 +174,7 @@ Adds a new event.
 
 **Note**:
 - `NAME` should only contain alphanumeric characters and spaces.
-- `RATE` must be a positive number.
+- `RATE` must be a positive number and below 1,000,000.
```

Comment:
Should `below 1,000,000` be in another sentence to be more consistent with the other commands in the UG?